Resources/Locale/en-US/store/categories.ftl

```diff
@@ -15,3 +15,7 @@ store-category-pointless = Pointless
 
 # Revenant
 store-category-abilities = Abilities
+
+# Wizard
+store-category-books = Books
+store-category-wands = Wands
```

Comment:
These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.